test/ered_SUITE.erl

```diff
@@ -225,16 +234,18 @@ t_client_crash(_) ->
     ?MSG(#{addr := {"127.0.0.1", Port}, master := true, msg_type := client_stopped}),
     ?MSG({'DOWN', _Mon, process, Pid0, crash}),
     ?MSG(#{msg_type := cluster_not_ok, reason := master_down}),
-    %% Instant error when client pid is dead. There's a race condition here. The
-    %% new client process can potentially come up fast and return "OK" to the
-    %% following commands.
-    {error, client_down} = ered:command(R, [<<"SET">>, <<"k">>, <<"v">>], <<"k">>),
+    %% Command immediately when the client process is dead. The cluster process
```

Comment:
```suggestion
    %% Connect immediately when the client process is dead. The cluster process
```

Reply:
No, this is `ered:command`, send command.

Reply:
```suggestion
    %% Send command when the client process is dead. The cluster process
```